pkg/gardenadm/cmd/connect/connect_test.go

```diff
@@ -1,41 +0,0 @@
-// SPDX-FileCopyrightText: SAP SE or an SAP affiliate company and Gardener contributors
```

Comment:
I assume that you deleted the test because `gardenadm connect` is hardly testable on its own, but it needs an integration test with a garden cluster. Is that understanding correct or did you have other reasons?